pjmedia/src/pjmedia-audiodev/alsa_dev.c

```diff
@@ -744,6 +787,12 @@ static pj_status_t open_playback (struct alsa_stream* stream,
         tmp_buf_size = tmp_period_size * 2;
     snd_pcm_hw_params_set_buffer_size_near (stream->pb_pcm, params,
                                             &tmp_buf_size);
+    if (result < 0) {
+        PJ_LOG (3,(THIS_FILE, "Unable to set period size: %d for "
+                  "playback device '%s', err: %s", (int)tmp_buf_size,
+                  stream->af->devs[param->play_id].name, snd_strerror(result)));
```

Comment:
Minor: perhaps better put "Warning: unable..." or "Ignored: unable...", to avoid confusion when investigating log file in the future?